editor/src/components/inspector/inspector-strategies/hug-contents-basic-strategy.ts

```diff
@@ -1,5 +1,6 @@
 import type { ElementPathTrees } from '../../../core/shared/element-path-tree'
```

Comment:
minor nitpick, but could you rename this file?

Reply:
done in https://github.com/concrete-utopia/utopia/pull/6087/commits/2731834981717d7df9403fc9f979a960cadf0043 and https://github.com/concrete-utopia/utopia/pull/6087/commits/917e7767d63d180f9d1dba0b3c5f21fa694109fd